### PR TITLE
community/gitea: fix missing a build tag for SQLite

### PR DESCRIPTION
**Pull request by typeless**
Fixes https://github.com/go-gitea/gitea/issues/6435

### CODE REVIEW

**Comment by andypost** [comment]
Related https://github.com/alpinelinux/aports/pull/6687

File: community/gitea/APKBUILD [hunk 1]
```diff
@@ -30,7 +30,7 @@ prepare() {
 build() {
         cd "$builddir"
 	export GOPATH="$srcdir"
-	export TAGS="tidb sqlite"
+	export TAGS="tidb sqlite sqlite_unlock_notify"
```
Comment (andypost) [inline community/gitea/APKBUILD]: as you affect build, it needs to bump pkgver

Comment (typeless) [reply community/gitea/APKBUILD]: Done.

**Comment by andypost** [comment]
@typeless please squash commits with `git rebase -i HEAD~2` it should be single commit

**Comment by typeless** [comment]
@andypost Done.

**Pull request closed without merging**